ga4gh/serverconfig.py

```diff
@@ -84,3 +86,29 @@ class TestOidcConfig(TestConfig):
     OIDC_AUTHZ_ENDPOINT = "https://accounts.example.com/auth"
     OIDC_TOKEN_ENDPOINT = "https://accounts.example.com/token"
     OIDC_TOKEN_REV_ENDPOINT = "https://accounts.example.com/revoke"
+
+
+class FlaskDefaultConfig(object):
```

Comment:
OK, the comments at least make it clear that this is what we're doing. Flask configuration really isn't all that amazing...